### ISSUE TITLE
Microsoft Copilot

### ISSUE THREAD

**Issue by almthor** [brave/brave-browser#34886]
Bing copilot allows me to ask only 4 question 
Normaly bing copilot lets you ask 4 question without logging in. when logged in with microsoft account it increases to 30.
But with brave even when I logged in it doesn't let me to ask more than 4. Tried with vivaldi and edge and didn't had this problem.

## Steps to Reproduce

   1. Open bing chat
   2. log in with microsoft account
   3. write some input

## Expected result:
Being able to enter 30 input.

## Reproduces how often: 
Every time

## Brave version (brave://version info)
[Version 1.61.101 Chromium: 120.0.6099.71 (Official Build) (64-bit)](https://brave.com/latest/)

**Comment by rebron** [comment]
@almthor Are you still seeing this behavior with Copilot using Brave? Our latest is 1.61.109, can you update to that version as well.

**Comment by almthor** [comment]
Hello, thank you for asking. I still have the same issue with the last version you mentioned both on windows and android.

**Comment by tarvcode** [comment]
I can confirm the same.  I've tried deleting cookies, resigning in, enable/disable shields and ublock, disabling pihole.  I'm signed in but it acts as though I am not signed in.  

On Version 1.61.120 now.

![image](https://github.com/brave/brave-browser/assets/16649815/2a3cf732-8bab-41dc-b837-e3a1ce519afb)

**Comment by almthor** [comment]
Yes I am still waiting for a fix :) Its weird that this only happens with brave.
